spec/requests/v1/vocabularies_spec.rb

```diff
@@ -124,15 +131,15 @@
     context 'when deleting vocabulary' do
       before do
         FactoryBot.create(:vocabulary)
-        delete '/api/v1/vocabularies/subjects'
+        delete '/api/v1/vocabularies/mythical_creatures'
       end
 
       it 'returns 204' do
         expect(response.status).to be 204
       end
 
       it 'removes vocabulary from database' do
-        expect(Vocabulary.find_by(string_key: 'subjects')).to be nil
+        expect(Vocabulary.find_by(string_key: 'mythical_creatures')).to be nil
```

Comment:
For this test, you may want to check for the presence of the mythical_creatures vocabulary before deleting it, just to prove that it actually existed before the delete succeeded.